cmd/spire-agent/cli/run/run.go

```diff
@@ -72,6 +75,11 @@ type agentConfig struct {
 	UnusedKeys []string `hcl:",unusedKeys"`
 }
 
+type sdsConfig struct {
+	SDSDefaultSVIDName   string `hcl:"sds_default_svid_name"`
+	SDSDefaultBundleName string `hcl:"sds_default_bundle_name"`
```

Comment:
Just noticed the stutter on the names here. How would you feel about removing the sds prefix since these are in an `sds` section?
```suggestion
	DefaultSVIDName   string `hcl:"default_svid_name"`
	DefaultBundleName string `hcl:"default_bundle_name"`
```